pymc/distributions/multivariate.py

```diff
@@ -1656,8 +1667,8 @@ def dist(
 
         cholesky = Cholesky(lower=True, on_error="raise")
 
-        if kwargs.get("size", None) is not None:
-            raise NotImplementedError("MatrixNormal doesn't support size argument")
+        # if kwargs.get("size", None) is not None:
```

Comment:
We can't to this change. If I remember correctly either our logp or rng_fn do not (yet) accomodate the size parameter. That's why that check was there.

On the other hand, that also means that your `get_moment` does not need to worry about it for tbe time being.

We should probably move this restriction to the RV itself, but that's a separate matter. CC @kc611

Reply:
Done.